lib/plugins/aws/deploy/lib/create-stack.js

```diff
@@ -88,6 +88,21 @@ module.exports = {
                 this.provider.disableTransferAccelerationForCurrentDeploy();
               }
             }
+
+            const inactiveStateNames = new Set(['REVIEW_IN_PROGRESS']);
+            const isSingleStack = data.Stacks.length <= 1;
+            const stackStatus = data.Stacks[0].StackStatus;
+
+            if (isSingleStack && inactiveStateNames.has(stackStatus)) {
+              const errorMessage = [
+                'Service cannot be deployed as the CloudFormation stack ',
+                "is in the 'REVIEW_IN_PROGRESS' state. ",
```

Comment:
So good. The `inactivateStateNames` set in the modified code can be expanded, so it would be appropriate to update the errorMessage output accordingly.